### PR TITLE
fix readme for inference

### PR DESCRIPTION
**Pull request by liangxhao**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

完善推理的readme，删除无用页面，并补充
①精度的说明
②MindOCR推理支持的模型列表
③默认通过lite进行推理

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: docs/cn/inference/models_list_cn.md [hunk 1]
```diff
@@ -2,21 +2,26 @@
...
-
+请在导出或下载已预先导出的MindIR文件，并参考[模型转换教程](./convert_tutorial_cn.md)，再进行推理。
 
 ### 1. 文本检测
 
-| 名称                        | 模型    | 骨干网络    | 配置文件                                                  | 参考&下载链接                                       |
-|:---------------------------|:--------|:----------|:---------------------------------------------------------|:-------------------------------------------------|
-| en_ms_det_dbnet_resnet50   | DBNet   | ResNet-50 | [yaml](../../../configs/det/dbnet/db_r50_icdar15.yaml)   | [dbnet](../../../configs/det/dbnet/README_CN.md) |
-| en_ms_det_dbnetpp_resnet50 | DBNet++ | ResNet-50 | [yaml](../../../configs/det/dbnet/db++_r50_icdar15.yaml) | [dbnet++](../../../configs/det/dbnet/README_CN.md) |
+| 模型     | 骨干网络     | 语言 | 配置文件                                                                                | 参考&下载链接                           |
```
Comment (HaoyangLee) [inline docs/cn/inference/models_list_cn.md:11]: 该表最后一列有必要写么

File: docs/cn/inference/convert_tutorial_cn.md [hunk 2]
```diff
@@ -93,6 +92,20 @@ N使用多个可选值，HW使用固定值，config.txt如下：
 
 为了简化模型转换流程，我们开发了**自动分档工具**，可以从数据集中统计选择动态值和模型转换，详细教程请参考[模型Shape分档](./convert_dynamic_cn.md)。
 
+#### 1.3 精度
```
[the remark this answers is not in the document]
Comment (liangxhao) [reply docs/cn/inference/convert_tutorial_cn.md]: 已修改

File: docs/cn/inference/convert_tutorial_cn.md [hunk 3]
```diff
@@ -93,6 +92,20 @@ N使用多个可选值，HW使用固定值，config.txt如下：
...
+
+对于模型推理的精度，需要转换模型时通过在`converter_lite`设置。请参考[Ascend转换工具功能说明](https://www.mindspore.cn/lite/docs/zh-CN/master/use/cloud_infer/converter_tool_ascend.html)，
+可以在上述`config.txt`的`[ascend_context]`中增加`precision_mode`参数来设置精度：
```
[the remark this answers is not in the document]
Comment (liangxhao) [reply docs/cn/inference/convert_tutorial_cn.md]: 已修改

File: docs/cn/inference/convert_tutorial_cn.md [hunk 4]
```diff
@@ -209,7 +222,26 @@ atc --model=det_db.onnx \
 
 为了简化模型转换流程，我们开发了**自动分档工具**，可以一键式完成动态值选择和模型转换过程，详细教程请参考[模型Shape分档](./convert_dynamic_cn.md)。
 
-#### 2.3 ONNX -> MindIR
+##### 2.3.2 精度
```
[the remark this answers is not in the document]
Comment (liangxhao) [reply docs/cn/inference/convert_tutorial_cn.md]: 已修改

File: docs/cn/inference/model_evaluation_cn.md [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
 		--gt_path=/path/to/rec_gt.txt \
 		--pred_path=/path/to/prediction/rec_results.txt \
 		--character_dict_path=/path/to/xxx_dict.txt
 ```
-请注意，character_dict_path是可选参数，默认字典仅支持英文识别。
 
-在评估PaddleOCR系列模型时，请参照[第三方模型支持列表](./models_list_thirdparty_cn.md)使用对应字典。
+请注意，character_dict_path是可选参数，默认字典仅支持数字和英文小写。
+
+在评估PaddleOCR/MMOCR系列模型时，请参照[第三方模型支持列表](./models_list_thirdparty_cn.md)使用对应字典。
```
[the remark this answers is not in the document]
Comment (liangxhao) [reply docs/cn/inference/model_evaluation_cn.md]: 已修改

File: docs/cn/inference/models_list_cn.md [hunk 6]
```diff
@@ -2,21 +2,26 @@
 
 ## 推理 - MindOCR模型推理支持列表
 
-
 MindOCR推理支持训练端ckpt导出的模型，本文档展示了已适配的模型列表。
 
-请在下载已预先导出的MindIR文件，并参考[模型转换教程](./convert_tutorial_cn.md)。
-
+请在导出或下载已预先导出的MindIR文件，并参考[模型转换教程](./convert_tutorial_cn.md)，再进行推理。
```
[the remark this answers is not in the document]
Comment (liangxhao) [reply docs/cn/inference/models_list_cn.md]: 已修改

**Review by jianyunchao — APPROVED** [review]

**Pull request merged**